bitbots_vision/src/bitbots_vision/vision_modules/yolo_handler.py

```diff
@@ -4,14 +4,12 @@
 import time
 try:
     from pydarknet import Detector, Image
-except:
-    rospy.logerr("Not able to run YOLO! Its only executable under python3 with yolo34py or yolo34py-gpu installed.")
+except ImportError:
+    rospy.logerr("Not able to run Darknet YOLO! Its only executable under python3 with yolo34py or yolo34py-gpu installed.")
```

Comment:
Why should it be agressive?
I think automation would not be useful because there are differences in speed etc. between opencv and darknet especially if you run the vision in a docker. It is a bit confusing if the performance of the vision depends on the installed dependencies and not the corresponding configuration. 
Also in docker I only get like 1 fps in darknet so I also want to use the opencv implementation also under python3. The main goal of this exception was to keep python 2 supported if the darknet implementation only exists for python 3.